homeassistant/components/hassio/repairs.py

```diff
@@ -22,7 +22,7 @@
 from .handler import async_apply_suggestion
 from .issues import Issue, Suggestion
 
-SUGGESTION_CONFIRMATION_REQUIRED = {"system_execute_reboot"}
+SUGGESTION_CONFIRMATION_REQUIRED = {"system_adopt_data_disk", "system_execute_reboot"}
```

Comment:
Side note: I think we need to add `"repairs"` as a dependency in the manifest.

Reply:
Hm good catch https://github.com/home-assistant/core/pull/115486